pkgs/development/python-modules/numpy/default.nix

```diff
@@ -106,10 +106,31 @@ in buildPythonPackage rec {
     typing-extensions
   ];
 
-  checkPhase = ''
+  checkPhase = let
+    # https://github.com/numpy/numpy/issues/24548
+    disabledTests = lib.optionals stdenv.isx86_32 [
```

Comment:
Whats the difference to isi686 which I would have used. see https://github.com/NixOS/nixpkgs/blob/master/lib/systems/inspect.nix#L20-L21

Reply:
Updated